eng/testing/tests.mobile.targets

```diff
@@ -101,14 +101,14 @@
     </RuntimeConfigParserTask>
 
     <MonoAOTCompiler Condition="'$(RunAOTCompilation)' == 'true'"
-        CompilerBinaryPath="$(MicrosoftNetCoreAppRuntimePackNativeDir)cross\$(PackageRID)\mono-aot-cross"
+        CompilerBinaryPath="$(MonoArtifactsPath)\cross\$(PackageRID)\mono-aot-cross"
         OutputDir="$(_MobileIntermediateOutputPath)"
         Mode="Full"
         OutputType="AsmOnly"
         Assemblies="@(AotInputAssemblies)"
         AotModulesTablePath="$(BundleDir)\modules.c"
         UseLLVM="$(MonoEnableLLVM)"
-        LLVMPath="$(MicrosoftNetCoreAppRuntimePackNativeDir)cross\$(PackageRID)">
+        LLVMPath="$(MonoArtifactsPath)\cross\$(PackageRID)">
```

Comment:
Yes, `opt` and `llc` are the other executables

Reply:
> Does this directory have any files other than `mono-aot-cross`? This becomes `--llvm-path=..` for `mono-aot-cross`, but I'm not sure what it is used for.

For wasm it only has mono-aot-cross because we use wasm-opt and emcc from the emscripten packages. (also the opt and llc that we were bundling here didn't support wasm)